web3.js/src/transaction.ts

```diff
@@ -490,7 +490,7 @@ export class Transaction {
   /**
    * Get the estimated fee associated with a transaction
    */
-  async getEstimatedFee(connection: Connection): Promise<number> {
+  async getFeeForMessage(connection: Connection): Promise<number> {
```

Comment:
Because this is live, we have little choice but to support the old name for a while, deprecated. If you landed this PR, anyone calling `getEstimatedFee` would find their software no longer compiles with TypeScript (at best) or works in production (at worst).

Rename it, but also create a new method with the old name, that just calls straight through to the one with the new name.

Reply:
Okay, will do!